ksql-engine/src/test/resources/query-validation-tests/group-by.json

```diff
@@ -128,10 +282,10 @@
       ]
     },
     {
-      "name": "group by ROWKEY",
+      "name": "ROWKEY (stream->table)",
```

Comment:
A rename like this messes up QueryTranslationTest's discovery of any existing expected topology files, but it's fine in this case since the group-by QTTs were added after 5.1, so there are no existing expected topology files for these tests as a result.

Reply:
Good to know!